src/File/FileFinder.php

```diff
@@ -23,6 +26,34 @@ public function __construct(
 	{
 	}
 
+	private function tryToMakeRelativeExcludePath(string $excludePath, string $inPath): ?string
```

Comment:
for https://github.com/symfony/symfony/blob/v6.2.12/src/Symfony/Component/Finder/Iterator/ExcludeDirectoryFilterIterator.php#L62